### PR TITLE
{tools}[GCCcore/6.3.0] Gtk-Doc v1.26, ITSTool v2.0.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2066.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0fef9ca3efeb95ea7d579bc4a73a7426 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2572.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/3382843b5da61fdee2aae54191e38987 for a full test report.

**Comment by boegel** [comment]
@easybuilders/easybuild-easyconfigs-maintainers please review (dep for #5044)

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
ca005.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1f214d8b91357ffd6a3b222f9a339db8 for a full test report.

File: easybuild/easyconfigs/g/Gtk-Doc/Gtk-Doc-1.26-GCCcore-6.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+source_urls = ['https://download.gnome.org/sources/gtk-doc/%(version_major_minor)s']
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['bff3f44467b1d39775e94fad545f050faa7e8d68dc6a31aef5024ba3c2d7f2b7']
+
+builddependencies = [
+    ('binutils', '2.27'),
+    ('ITSTool', '2.0.2'),
+]
+
```
Comment (verdurin) [inline easybuild/easyconfigs/g/Gtk-Doc/Gtk-Doc-1.26-GCCcore-6.3.0.eb]: @boegel I suspect some more dependencies are required:
```
checking for dblatex... no
checking for fop... no
configure: WARNING: neither dblatex nor fop found, so no pdf output from xml
checking for XML catalog... not found
checking for xmlcatalog... /usr/bin/xmlcatalog
checking for DocBook XML DTD V4.3 in XML catalog... not found
configure: error: could not find DocBook XML DTD V4.3 in XML catalog
 (at easybuild/tools/run.py:444 in parse_cmd_output)
```
is the build failure I saw.

Comment (boegel) [inline easybuild/easyconfigs/g/Gtk-Doc/Gtk-Doc-1.26-GCCcore-6.3.0.eb]: Hmm, looks like it needs http://docbook.org/xml/4.3/ + libxml (for `xmlcatalog`).

Do you have any `docbook` RPMs installed @verdurin?

```
$ rpm -qa | grep docbook
docbook-style-xsl-1.78.1-3.el7.noarch
docbook-dtds-1.0-60.el7.noarch
```

Comment (boegel) [inline easybuild/easyconfigs/g/Gtk-Doc/Gtk-Doc-1.26-GCCcore-6.3.0.eb]: @verdurin Are you still seeing the same problem with `libxml2` included as a build dep?

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (2 easyconfigs in this PR)
ca005.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/245ba186de9e49a0f5a1604862bea3dc for a full test report.

File: easybuild/easyconfigs/g/Gtk-Doc/Gtk-Doc-1.26-GCCcore-6.3.0.eb (hunk 1 above)
Comment (verdurin) [inline easybuild/easyconfigs/g/Gtk-Doc/Gtk-Doc-1.26-GCCcore-6.3.0.eb]: @boegel Here's the build failure with the new dep:
```
checking for dblatex... no
checking for fop... no
configure: WARNING: neither dblatex nor fop found, so no pdf output from xml
checking for XML catalog... not found
checking for xmlcatalog... /camp/apps/eb/review/software/libxml2/2.9.4-GCCcore-6.3.0/bin/xmlcatalog
checking for DocBook XML DTD V4.3 in XML catalog... not found
configure: error: could not find DocBook XML DTD V4.3 in XML catalog
 (at easybuild/tools/run.py:444 in parse_cmd_output)
```

Comment (boegel) [inline easybuild/easyconfigs/g/Gtk-Doc/Gtk-Doc-1.26-GCCcore-6.3.0.eb]: cfr. https://trac.macports.org/ticket/26151

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
ca003.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/11f582bc70beaac841bd42578b341635 for a full test report.

**Comment by verdurin** [comment]
Still fails for me:
```
checking for xsltproc... /usr/bin/xsltproc
checking for dblatex... no
checking for fop... no
configure: WARNING: neither dblatex nor fop found, so no pdf output from xml
checking for XML catalog... not found
checking for xmlcatalog... /camp/apps/eb/review/software/libxml2/2.9.4-GCCcore-6.3.0/bin/xmlcatalog
checking for DocBook XML DTD V4.3 in XML catalog... not found
configure: error: could not find DocBook XML DTD V4.3 in XML catalog
 (at easybuild/tools/run.py:444 in parse_cmd_output)
```

**Comment by migueldiascosta** [comment]
@boegel #5504 was closed as "No longer relevant due to old toolchain", same here?

**Comment by boegel** [comment]
I must have needed this as a dep for something, but it hasn't popped up since, so closing indeed...

**Pull request closed without merging**